tests/functional/monitoring/libtest/test_workload_fixture.py

```
@@ -135,7 +208,9 @@ def test_workload_rbd_in_some_other_way(workload_storageutilization_50p_rbd):
     this and the previous test are using the same workload. You can check this
     by plotting ``ceph_osd_stat_bytes_used`` value via OCP Prometheus.
     """
-    logger.info(workload_storageutilization_50p_rbd)
+    logger.info("Starting test: Demonstrate reuse of RBD workload fixture")
+    logger.info(f"Workload fixture data: {workload_storageutilization_50p_rbd}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)